apps/crn-server/src/routes/research-outputs.route.ts

```diff
@@ -134,7 +136,11 @@ export const researchOutputRouteFactory = (
 
       // TODO: update the published value in hasEditResearchOutputPermission in
       // display draft task. Currently we are not sending the value published
-      if (!loggedInUser || !hasEditResearchOutputPermission(userRole, false)) {
+
+      if (
+        !loggedInUser ||
+        !hasEditResearchOutputPermission(userRole, publish)
```

Comment:
This needs to be the value of `published` to check if the research output is published or not 😉 For example, if the RO is already published and the user is just a member, `hasEditResearchOutputPermission` should give false. 
The `publish` that it's being passed here now is to publish the RO. I think we're not sending this `published` value to the BE yet, but we need to pass it so we can call this `hasEditResearchOutputPermission` :)

```suggestion
        !hasEditResearchOutputPermission(userRole, published)
```